src/composables/useNetwork.ts

```diff
@@ -26,9 +38,6 @@ export const isTestnet = computed(() => isKovan.value || isGoerli.value);
 /**
  * METHODS
  */
-export function setNetworkId(id: Network) {
-  networkId.value = id;
-}
```

Comment:
Should we not keep the possibilty to change the networkId programmatically? Ideally we have a single source of truth for what the current app network is, e.g. `networkId`. When a user lands on a route it's set to the relevant network, when a user switches network with the network switcher it's set to the relevant network. Then eventually we can transition everything to be reactive so we don't need refreshes.

Reply:
Do you mean should we keep this in preparation for future changes? Currently, there's no point for networkId to be reactive as network is always constant after an app is loaded.

Reply:
Yeah I guess that's what I was getting at, but actually, I think it's fine to remove it for now. We should focus on getting this working with a page reload first.